cmd/controller/main.go

```diff
@@ -159,6 +161,7 @@ func main() {
 	flag.StringVar(&nodeNoProxy, "node-no-proxy", ".svc,.cluster.local,localhost,127.0.0.1", "If set, it configures the 'NO_PROXY' environment variable on the nodes.")
 	flag.StringVar(&nodeInsecureRegistries, "node-insecure-registries", "", "Comma separated list of registries which should be configured as insecure on the container runtime")
 	flag.StringVar(&nodeRegistryMirrors, "node-registry-mirrors", "", "Comma separated list of Docker image mirrors")
+	flag.StringVar(&nodeMaxLogSize, "node-max-log-size", "", "Maximum size for docker logfile")
```

Comment:
We should have a default value here rather than requiring the parameter to be passed.

Reply:
Disagree, the empty default value ensures, that this is not a breaking change and when you do not pass it, the behaviour is as it is now.
This will make it more likely to be accepted upstream.

Reply:
OK, makes sense. Are you gonna do the modification?

Reply:
Yes, will work on it today